integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -209,17 +209,21 @@ case class CarbonLoadDataCommand(
     * 4. Session property CARBON_OPTIONS_SORT_SCOPE
     * 5. Default Sort Scope LOAD_SORT_SCOPE
     */
-    if (tableProperties.get(CarbonCommonConstants.SORT_COLUMNS) != null &&
-        tableProperties.get(CarbonCommonConstants.SORT_SCOPE) == null) {
-      // If there are Sort Columns given for the table and Sort Scope is not specified,
-      // we will take it as whichever sort scope given or LOCAL_SORT as default
-      optionsFinal
-        .put(CarbonCommonConstants.SORT_SCOPE,
-          carbonProperty
-            .getProperty(
-              CarbonLoadOptionConstants.CARBON_TABLE_LOAD_SORT_SCOPE + table.getDatabaseName + "." +
-              table.getTableName, carbonProperty.getProperty(CarbonCommonConstants.LOAD_SORT_SCOPE,
-                SortScopeOptions.getSortScope("LOCAL_SORT").toString)))
+    if (tableProperties.get(CarbonCommonConstants.SORT_COLUMNS) == null ||
+        tableProperties.get(CarbonCommonConstants.SORT_COLUMNS).equals("")) {
+      // If tableProperties.SORT_COLUMNS is null
+      optionsFinal.put(CarbonCommonConstants.SORT_SCOPE, "NO_SORT")
+    }
+    else if (tableProperties.get(CarbonCommonConstants.SORT_SCOPE) == null ||
+             tableProperties.get(CarbonCommonConstants.SORT_SCOPE).equals("")) {
+      // If tableProperties.SORT_COLUMNS is not null
+      // and tableProperties.SORT_SCOPE is null
+      optionsFinal.put(CarbonCommonConstants.SORT_SCOPE,
+        carbonProperty.getProperty(CarbonLoadOptionConstants.CARBON_TABLE_LOAD_SORT_SCOPE +
+          table.getDatabaseName + "." + table.getTableName,
+          carbonProperty.getProperty(CarbonLoadOptionConstants.CARBON_OPTIONS_SORT_SCOPE,
+            carbonProperty.getProperty(CarbonCommonConstants.LOAD_SORT_SCOPE,
+              "LOCAL_SORT"))))
```

Comment:
why is LOAD_SORT_SCOPE_DEFAULT changed into "NO_SORT" in CarbonCommonConstants and this default value is also "LOCAL_SORT"
the ddl doc also describe that "NO_SORT" is the default value

Reply:
If the SORT_COLUMNS are set and any SORT_SCOPE is not found, then LOCAL_SORT is used.
If no SORT_COLUMN are set, then take the SORT_SCOPE as NO_SORT.

The default behaviour is NO_SORT if SORT_COLUMNS are not provided, and LOCAL_SORT if SORT_COLUMNS are provided explicetely in CREATE TABLE command. If nothing is specified in CREATE TABLE command, it takes SORT_SCOPE as NO_SORT and SORT_COLUMNS as none.

Reply:
please avoid using the plain string here

Reply:
About the behavior you mentioned above, is it exactly the doc trying to describe? If not, please optimize the doc in this PR to ensure the modifications' integrity.

Reply:
besides, please check the comments of this method and ensure its correctness after your modification.